Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.23 AS builder
+
+WORKDIR /app
+
+COPY go.mod go.sum ./
+
+RUN go mod download
+
+COPY . .
+
+RUN go build -o /app/cmd/bff/cmd ./cmd/bff
```

Comment:
what modules would you like to add, and keep as entrypoint?

Reply:
let's keep out put binary as entry point and server command as entry point